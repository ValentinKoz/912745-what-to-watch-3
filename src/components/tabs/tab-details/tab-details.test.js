import React from "react";
import renderer from "react-test-renderer";
import TabDetails from "./tab-details.jsx";

const film = {
  genre: `action`,
  releaseDate: 1998,
  poster: ``,
  bgPoster: ``,
  ratingScore: ``,
  ratingLevel: ``,
  ratingCount: 0,
  text: ``,
  director: `Steven Zaillian`,
  runTime: `1h 39m`,
  starring: `John Travolta, Robert Duvall, Stephen Fry`,
  reviews: []
};

it(`Should TabDetails renderer correctly`, () => {
  const tree = renderer
    .create(<TabDetails
      film = {film}
    />).toJSON();

  expect(tree).toMatchSnapshot();
});
