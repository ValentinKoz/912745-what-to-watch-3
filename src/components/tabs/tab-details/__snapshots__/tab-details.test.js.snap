// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should TabDetails renderer correctly 1`] = `
<div
  className="movie-card__text movie-card__row"
>
  <div
    className="movie-card__text-col"
  >
    <p
      className="movie-card__details-item"
    >
      <strong
        className="movie-card__details-name"
      >
        Director
      </strong>
      <span
        className="movie-card__details-value"
      >
        Steven Zaillian
      </span>
    </p>
    <p
      className="movie-card__details-item"
    >
      <strong
        className="movie-card__details-name"
      >
        Starring
      </strong>
      <span
        className="movie-card__details-value"
      >
        John Travolta,
        <br />
         Robert Duvall,
        <br />
         Stephen Fry,
      </span>
    </p>
  </div>
  <div
    className="movie-card__text-col"
  >
    <p
      className="movie-card__details-item"
    >
      <strong
        className="movie-card__details-name"
      >
        Run Time
      </strong>
      <span
        className="movie-card__details-value"
      >
        1h 39m
      </span>
    </p>
    <p
      className="movie-card__details-item"
    >
      <strong
        className="movie-card__details-name"
      >
        Genre
      </strong>
      <span
        className="movie-card__details-value"
      >
        action
      </span>
    </p>
    <p
      className="movie-card__details-item"
    >
      <strong
        className="movie-card__details-name"
      >
        Released
      </strong>
      <span
        className="movie-card__details-value"
      >
        1998
      </span>
    </p>
  </div>
</div>
`;
